src/InstanceManager.ts

```diff
@@ -656,6 +656,11 @@ export default class InstanceManager {
       // _iresearch_analyzers is available only since 3.4
       systemCollections.delete('_iresearch_analyzers');
     }
+    if (version.major <= 3 && version.minor > 4) {
+      // _iresearch_analyzers is called _analyzers since 3.5
+      systemCollections.delete('_iresearch_analyzers');
+      systemCollections.add('_analyzers');
+    }
```

Comment:
Please instead update the `Set` construction above to the most current variant, and here add exceptions for older versions (which we then can just delete at some point).